quickstarts/aws/amazon-inspector/config.yml

```diff
@@ -0,0 +1,29 @@
+slug: amazon-inspector
+description: |-
+  ## What is Amazon Inspector?
+  Amazon Inspector is a vulnerability management service that continuously scans your AWS workloads for software vulnerabilities and unintended network exposure. Amazon Inspector automatically discovers and scans running Amazon EC2 instances, container images in Amazon Elastic Container Registry (Amazon ECR), and AWS Lambda functions for known software vulnerabilities and unintended network exposure.
+  ### Get started!
+  Start monitoring Amazon Inspector by connecting Amazon Web Services (AWS) to New Relic! Take a look at our documentation for information on how to set up this integration.
+  ### More info
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-inspector-monitoring-integration/) to learn more about New Relic monitoring for Amazon Inspector.
+summary: |-
+  Monitor Amazon Inspector by connecting AWS to New Relic
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: Amazon Inspector
+documentation:
+  - name: Amazon Inspector installation docs
+    description: |
+      Monitor Amazon Inspector by connecting AWS to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-inspector-monitoring-integration/
+keywords:
+  - aws
+  - amazon web services
+  - inspector
+dashboards:
+  - amazon-inspector
+dataSourceIds:
+  - amazon-cloudwatch-metric-streams
```

Comment:
alert-policies are added to the file changes. Could you please attach the alert-policy to this config.yml file.

Reply:
I have added alertPolicies.

cc - @pkosanam @mdumpati